### ISSUE TITLE
Can't type or see content in WYSIWYG field visual mode with options when block editor is active (Firefox only)

### ISSUE THREAD

**Issue by lumpysimon** [CMB2/CMB2#1519]
When all the following circumstances are true, it's not possible to type in the WYSIWYG field or see its content:

 - Using Firefox browser (on Mac or Windows),
 - The WYSIWYG field is on a post type that has the block editor enabled,
 - Any settings are passed in via the `options` array (it doesn't matter what they are),
 - You are viewing the field in Visual mode.

If it's a post type where the block editor is disabled, or the `options` array is not passed in, then everything is fine.

It all works fine in Chrome and Safari, this issue only happens in Firefox (on Mac and Windows).

In visual mode the field looks empty, and the editor toolbar buttons are not correctly rendered:

<img width="821" alt="Image" src="https://github.com/user-attachments/assets/d895516b-25bb-42eb-87b7-30d0527501d5" />

In text mode, the content is visible:

<img width="824" alt="Image" src="https://github.com/user-attachments/assets/6d031f8c-0d83-44ae-8130-c0e807973ffc" />

**Comment by tw2113** [comment]
Possible that there's some CSS of some sort affecting things.

**Comment by Ipstenu** [comment]
I can reproduce this on Safari/Mac.

Console:

`TypeError: null is not an object (evaluating 'n.setBaseAndExtent')`

and

`Unhandled Promise Rejection: [object Object]`

Weirdly _sometimes_ it works just fine. If I flip to code, add content, safe, and refresh, it magically works. 

I'd be surprised if anyone has the same CSS as I do :D

**Comment by lumpysimon** [comment]
Yes the same for me: it doesn't work, but switching to code mode, adding content, etc, then it (sometimes) does.

I'm pretty certain it's nothing to do with CSS.

**Comment by Ipstenu** [comment]
I was able to narrow this down to me using GRIDS! As soon as I moved the problematic editor blocks out of the grid, they worked perfectly.

**Comment by rawsta** [comment]
I got the same problem with the TinyMCE. 
The only workaround that kind of works is from the Core ticket (https://core.trac.wordpress.org/ticket/62450)
"This only happens when the editor is initializing on the Visual tab. If I switch to the Text tab and then reload the page, the editor initializing normally."
